pulp-docker.spec

```diff
@@ -2,13 +2,13 @@
 %{!?python_sitearch: %global python_sitearch %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(1))")}
 
 # The release number
-%global release_number 1
+%global release_number 0.1.beta
```

Comment:
Could you explain this release_number beta format?

Reply:
Pulp rpms use the release field to set if it's alpha/beta/rc/ga.  all pre-GA release numbers use a 0.n.stage so each subequent stage can take precedent over the previous untill GA, which gets a '1'